src/AutoPilotPlugins/PX4/RadioComponent.cc

```diff
@@ -140,7 +140,11 @@ QStringList RadioComponent::paramFilterList(void) const
 
 QWidget* RadioComponent::setupWidget(void) const
 {
-    return new PX4RCCalibration;
+    QGCQmlWidgetHolder* holder = new QGCQmlWidgetHolder();
```

Comment:
I think in general auto hides too much of the details of the code. So I stay away from it.